.github/workflows/ci-conan-clang.yml

```diff
@@ -19,17 +20,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [13.4.0]
+        version: [13.4.0, 13.5.0]
         compiler_version: [9, 10, 11]
         build_type: [Debug, Release]
         os: [ubuntu-20.04]
-        configs: ["True", "False"]
-        enable_double:  ["True", "False"]
+        configs: ['True']  # We don't build FhSim with clang
```

Comment:
Kunne hatt et bedre navn enn configs? Eksempelvis "use_double"?

Reply:
For clang we don't build with `enable_double=True`. We could have skipped the matrix item, but maybe in the future we will add meaningful configurations, like in the case msvc and gcc